utils/wasm-gen/src/generator/syscalls/invocator.rs

```diff
@@ -823,6 +831,29 @@ impl<'a, 'b> SyscallsInvocator<'a, 'b> {
         }
     }
 
+    fn store_waited_message_id(&self, instructions: &mut Vec<Instruction>) {
+        let Some(gr_message_id_indexes_handle) = self
+            .syscalls_imports
+            .get(&InvocableSyscall::Loose(SyscallName::MessageId))
+            .map(|&(_, call_indexes_handle)| call_indexes_handle as u32)
+        else {
+            // We automatically enable the `message_id` syscall import if the `wait` syscall is enabled in the config.
```

Comment:
I would panic here, because of not having `gr_message_id` when it's required to be in case of inserted `gr_wait*` functions.